### PR TITLE
OpenStack-Permissions-01

### PR DESCRIPTION
**Pull request by resmaa**
1. CLI Gateway Creation
2. Permission set OpenStack

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/415#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @resmaa. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: packages/docs/content/04-clusters/1-new-clusters.md
Comment (arvind-sys) [inline packages/docs/content/04-clusters/1-new-clusters.md:2269]: Not Necessary.
* Machine having Docker Engine Running

File: packages/docs/content/04-clusters/1-new-clusters.md
Comment (arvind-sys) [inline packages/docs/content/04-clusters/1-new-clusters.md:2279]: Image: gcr.io/spectro-images-public/release/spectro-installer:v1.0.1

File: packages/docs/content/04-clusters/1-new-clusters.md
Comment (arvind-sys) [inline packages/docs/content/04-clusters/1-new-clusters.md:2337]: /tmp/install-pcg-openstack-pcg-<unique-ID>

File: packages/docs/content/04-clusters/1-new-clusters.md
Comment (arvind-sys) [inline packages/docs/content/04-clusters/1-new-clusters.md:2345]: One Step Above
* Move the PCG.yaml file to /tmp folder for easy Execution

File: packages/docs/content/04-clusters/1-new-clusters.md
Comment (arvind-sys) [inline packages/docs/content/04-clusters/1-new-clusters.md:2360]: Not required. As we have already set. Remove this point "Once the deployment is completed, set the number of nodes as either 1 or 3."

**Review by arvind-sys — CHANGES_REQUESTED** [review]

**Pull request closed without merging**
